index.html

```diff
@@ -31,8 +49,23 @@ <h3>Add a new book</h3>
 				<button type="button" id="add-button">Add</button>
 				<div id="error"></div>
```

Comment:
[OPTIONAL] - We can consider using `type="submit"` on the button tag so that when the `enter` key is pressed, the form submits automatically and that will improve the user experience on your website👍